app/graphql/mutations/stories/update_story.rb

```diff
@@ -13,7 +13,10 @@ class UpdateStory < Mutations::BaseMutation
 
       def resolve(attributes)
         begin
+          require "pry"; binding.pry
```

Comment:
should probably get this pry out of here. I might even do a cmd+shft+f and see if there are any binding.pry hiding about in the code. We will pick all these blasted nits!